packages/next-auth/src/providers/united-effects.ts

```diff
@@ -0,0 +1,29 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface UnitedEffectsProfile extends Record<string, any> {
+  sub: string
+  email: string
+}
+
+export default function UnitedEffects<P extends UnitedEffectsProfile>(
+  options: OAuthUserConfig<P>
```

Comment:
If `issuer` is an option you should mark it here like so:
```suggestion
  options: OAuthUserConfig<P> & { issuer: string } 
```